### ISSUE TITLE
Better US congressional district polygons

### ISSUE THREAD

**Issue by bcamper** [whosonfirst-data/whosonfirst-data#634]
Looking at some of our US congressional district polygons, they are rough.

For example, the Alaska district extends into the water (however, this is also true of the Alaska state shape itself):
https://github.com/whosonfirst-data/whosonfirst-data-constituency-us/blob/master/data/110/873/735/7/1108737357.geojson

There are some higher quality congressional district polygon datasets we could use though:
https://github.com/benbalter/congressional-districts

Alaska from the latter dataset for comparison:
https://github.com/benbalter/congressional-districts/blob/gh-pages/AK/00.geojson

**Comment by nvkelso** [comment]
The US Census generally only releases these at 1:500,000 scale – they will always be a little "rough" from that source. 

I think you're responding to the shape of the feature not being along the shoreline (instead buffering out into the water). Best "display" polygon and related work is tracked under: https://github.com/whosonfirst-data/whosonfirst-data/issues/367.

Looks like the 114th shapes are out now:

- http://www.census.gov/geo/maps-data/data/cbf/cbf_cds.html

**Comment by nvkelso** [comment]
See also:

- https://github.com/whosonfirst/whosonfirst-cookbook/blob/master/how_to/creating_alt_geometries.md

**Comment by bcamper** [comment]
Yes, primarily a display issue with buffering around water I think.

**Comment by stepps00** [comment]
Suggesting that we take a similar approach as we've done [here](https://github.com/whosonfirst-data/whosonfirst-data/pull/539) and create new `alt-geom` features for constituencies in the US. In a similar, previous issue, we retained the default `region` geometry for each state, but added a "clipped" alt-geometry for each state with a water boundary (see: [Alaska](https://github.com/whosonfirst-data/whosonfirst-data/blob/master/data/856/885/73/85688573-alt-uscensus-display-terrestrial-zoom-10.geojson)). The alt-geom filename would be as follows:

`"wofid"-alt-uscensus-display-terrestrial-zoom-10.geojson`

The previous issue dealt with the same US Census data as outlined in this issue.

**Comment by nvkelso** [comment]
I'm fine with this approach – but it's not super high priority for me.

**Comment by stepps00** [comment]
Tracking new constituency alt-geom work in the `whosonfirst-data-constituency-us` repo ([here](https://github.com/whosonfirst-data/whosonfirst-data-constituency-us/compare/stepps00/constituency-alt-geoms)).

New 500k resolution alt-geometries for the `us-senate` features are in [this commit](https://github.com/whosonfirst-data/whosonfirst-data-constituency-us/commit/b39cc39faae450f1ff3be9e7de7d21125af1d006).
